core/src/main/java/smashdudes/ecs/systems/MovementSystem.java

```diff
@@ -23,6 +23,14 @@ public void updateEntity(Entity entity, float dt)
         p.position.x += v.velocity.x * dt;
         p.position.y += v.velocity.y * dt;
 
-        v.velocity.x *= v.deceleration * dt;
+        float decelerationThisFrame = v.deceleration * dt;
+        if (Math.abs(v.velocity.x) > decelerationThisFrame)
+        {
+            v.velocity.x -= Math.signum(v.velocity.x) * decelerationThisFrame;
+        }
+        else
+        {
+            v.velocity.x = 0;
+        }
```

Comment:
This is the new deceleration.  Before we were doing a multiply (which didnt work for all values,  if deceleration was greater than 1/dt it accelerated).

Now we subtract the deceleration value each frame from  the velocity, leading to a nice smooth deceleration that is easily tunable in boxtool